packages/threat-composer/src/components/workspaces/EditWorkspace/index.tsx

```diff
@@ -29,17 +29,29 @@ export interface EditWorkspaceProps {
   setVisible: React.Dispatch<React.SetStateAction<boolean>>;
   onConfirm: (workspace: string) => Promise<void>;
   value?: string;
-  editMode?: boolean;
+  editMode: 'add' | 'update' | 'clone';
   currentWorkspace?: Workspace;
   workspaceList: Workspace[];
   exampleWorkspaceList: Workspace[];
 }
 
+const BUTTON_LABEL = {
+  add: 'Add',
+  update: 'Update',
+  clone: 'Clone',
```

Comment:
For consistency with other labels such as "Add new workspace", should we make this one "Clone current workspace"? as while we are at it also change "Import" to "Import into current workspace"